### PR TITLE
Batch query quotenet's autocomplete for stock names

### PR DESCRIPTION
**Pull request by thedod**
See #17

### CODE REVIEW

File: sample-data/quotenet-output.csv [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿TEVA PFD 7,Teva Pharmaceutical Industries Ltd 7 % Conv.Cum.Pfd Shs Mandatory Conv. 15.12.2018,IL0062905489
```
Comment (nirgn975) [inline sample-data/quotenet-output.csv]: @thedod Why are we commit the output data?

Comment (thedod) [reply sample-data/quotenet-output.csv]: It's the simplest way I've found to document the output structure/semantics.
It's what Lior was looking for during the hackathon. Not too long and contains a few "not found"s.

I could make input + output examples much shorter (1-2 results + 1 "not found"). @nirgn975 Would you like me to do that?

Comment (nirgn975) [reply sample-data/quotenet-output.csv]: Yeah, keep it as short as possible but more importantly as informant and true.
(If you think this length is right, it's fine by me)

File: sample-data/quotenet-queries.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+TARO
```
Comment (thedod) [inline sample-data/quotenet-queries.txt:1]: multiple result

File: sample-data/quotenet-queries.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+TEVA PFD 7
```
Comment (thedod) [inline sample-data/quotenet-queries.txt:2]: single result

File: sample-data/quotenet-queries.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+MTIאם טי איי MWE.L
```
Comment (thedod) [inline sample-data/quotenet-queries.txt:3]: not found

**Review by nirgn975 — APPROVED** [review]

**Pull request merged**